packages/fxa-auth-server/lib/geodb.js

```diff
@@ -5,7 +5,7 @@
 'use strict';
 
 const config = require('../config').get('geodb');
-const geodb = require('../../fxa-geodb')(config);
+const geodb = require('fxa-geodb')(config);
```

Comment:
How is this working? Does node know to resolve this somewhere (like fxa-react), and if so can this change be extended to the other relative require statements?

Reply:
it's resolve through package.json using `"fxa-geodb": "workspace:*"` and yes we should be using this style everywhere. idk why i didn't catch them all with my grep